server/data/data-postgres/pom.xml

```diff
@@ -155,9 +155,7 @@
                 <artifactId>openjpa-maven-plugin</artifactId>
                 <version>${apache.openjpa.version}</version>
                 <configuration>
-                    <includes>org/apache/james/sieve/jpa/model/JPASieveQuota.class,
-                        org/apache/james/sieve/jpa/model/JPASieveScript.class,
-                        org/apache/james/user/jpa/model/JPAUser.class,
+                    <includes>org/apache/james/sieve/postgres/model/JPASieveScript.class,
```

Comment:
At the very least I want to see a Github issue for removing that JPA entity...

Reply:
cf https://github.com/linagora/james-project/issues/4928#issuecomment-1829281340